push-build.sh

```diff
@@ -155,9 +158,11 @@ fi
 GCS_DEST="devel"
 ((FLAGS_ci)) && GCS_DEST="ci"
 GCS_DEST=${FLAGS_release_type:-$GCS_DEST}
-GCS_DEST+="$FLAGS_gcs_suffix"
+# shellcheck disable=SC2154
```

Comment:
Can we explain why the disable is needed?

Reply:
I'm disabling any variable that gets declared outside of the script. I couldn't really think of a better way (when I defined the variable as ${VARIABLE_NAME:?}, for example, it broke the script on test and it was found to be dangerous.

Reply:
sorry, I mean in a comment in the file, it's nicer for maintainers down the line to understand why these are here

Reply:
Oh I see. For sure, will do